src/ray/raylet/worker_pool.cc

```diff
@@ -155,7 +158,7 @@ bool WorkerPool::DisconnectWorker(std::shared_ptr<Worker> worker) {
 // Protected WorkerPool methods.
 void WorkerPool::AddStartedWorker(pid_t pid) { started_worker_pids_.insert(pid); }
 
-uint32_t WorkerPool::NumStartedWorkers() const { return started_worker_pids_.size(); }
+int WorkerPool::NumStartedWorkers() const { return started_worker_pids_.size(); }
```

Comment:
Why change this to an `int`?

Reply:
See https://google.github.io/styleguide/cppguide.html, "On Unsigned Integers"

In this case it was necessary to avoid a signed/unsigned mismatch.

In general it's better to use signed ints, unless there is a good reason not to (i.e. for doing bit manipulations or modular arithmetic).